app/controllers/images_controller.rb

```
@@ -1,6 +1,18 @@
 # Basic images controller
 class ImagesController < ApplicationController
   def index
-    @hello_world = 'Hello World!'
+    @hello_world = "Hello World!"
+    @image = Image.new
+  end
+
+  def create
+    @image = Image.create(create_params)
```

You should use `create!` here so that it will raise an exception (instead of just returning `false`) if the creation fails.